samples/server/petstore/java-play-framework/build.sbt

```diff
@@ -6,10 +6,5 @@ lazy val root = (project in file(".")).enablePlugins(PlayJava)
 
 scalaVersion := "2.11.7"
 
-libraryDependencies ++= Seq(
-javaJdbc,
-cache,
-javaWs,
-"org.webjars" % "swagger-ui" % "2.2.10-1",
-"javax.validation" % "validation-api" % "1.1.0.Final"
-)
+libraryDependencies += "org.webjars" % "swagger-ui" % "2.2.10-1"
```

Comment:
Like I said in my previous comment, this will only work because the default value for `useSwaggerUI` and `useBeanValidation` is true. It will crash if the user change the default value.